.env.development

```
@@ -0,0 +1,6 @@
+GITHUB_ID=gh_client_id_placeholder
+GITHUB_SECRET=gh_client_secret_placeholder
+GOOGLE_ID=google_client_id_placeholder
+GOOGLE_SECRET=google_client_secret_placeholder
+NEXTAUTH_SECRET=R2xvYmFsRmluYW5jaWFsUGxhdGZvcm1TZWNyZXQyMDI0
+DATABASE_URL=postgresql://postgres@localhost:5432/Buckalew?schema=public
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)